lib/constants/ftu_constants.dart

```diff
@@ -20,29 +19,45 @@ const String onboardingPartThreeText =
 const String onboardingPartThreeButtonText = 'Let\'s get started';
 const String letsGetStartedButtonText = 'Let\'s get started';
 
-// Onboarding Screen Config
-const double imageHeight = 150.0; 
-var onboardingScreensConfig = [
+// Animator keys
+GlobalKey<AnimatorWidgetState> onboardingImagePartOneKey
+  = GlobalKey<AnimatorWidgetState>();
+GlobalKey<AnimatorWidgetState> onboardingImagePartTwoKey
+  = GlobalKey<AnimatorWidgetState>();
+GlobalKey<AnimatorWidgetState> onboardingImagePartThreeKey
+  = GlobalKey<AnimatorWidgetState>();
+  GlobalKey<AnimatorWidgetState> onboardingTextPartOneKey
+  = GlobalKey<AnimatorWidgetState>();
+GlobalKey<AnimatorWidgetState> onboardingTextPartTwoKey
+  = GlobalKey<AnimatorWidgetState>();
+GlobalKey<AnimatorWidgetState> onboardingTextPartThreeKey
+  = GlobalKey<AnimatorWidgetState>();
+
+// Onboarding Steps Config
+List<Map<String, dynamic>> onboardingStepsConfig = [
   {
     "text": onboardingPartOneText,
-    "image": Image.asset('images/onboarding/ano_standing.png', height: imageHeight),
-    "nextScreen": (context) => OnboardingScreens(screenNum: 1), 
+    "image": 'images/onboarding/ano_standing.png',
```

Comment:
I decided to just pass the image path so we can set the image width depending on the screen width as well!